src/annotator/components/AdderToolbar.js

```diff
@@ -18,17 +70,20 @@ function ToolbarButton({ badgeCount, icon, label, onClick, shortcut }) {
 
   return (
     <LabeledButton
-      classes="LabeledIconButton AdderToolbar__button"
+      className={classnames(
+        'flex flex-col gap-y-1 items-center py-2.5 px-2',
+        'text-annotator leading-none text-grey-7',
```

Comment:
Thanks for flagging this. I've mentioned in other comments before that Tailwind's `text-` prefixed generated classnames can be non-obvious. I ended up changing some color variable names at one point because of this, e.g. 

I will try to brainstorm a better variable name for the `annotator` font size.